Draw/Audio_VLC/Audio_VLC.py

```diff
@@ -1,49 +1,49 @@
 # -*- coding: utf-8 -*-
-# DO NOT EDIT THIS FILE
-# This file was automatically generated by Natron PyPlug exporter version 10.
+# DON'T NOT EDIT THIS FILE
```

Comment:
I put that in on purpose in case anyone viewing this file might've worked with PyPlug before.  I was trying to communicate that the advice that was put there previously should be ignored.